volume/drivers/coprhd/scaleio.go

```diff
@@ -0,0 +1,40 @@
+package coprhd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	diskById = "/dev/disk/by-id"
+)
+
+// getScaleIoDevice returns the local device associated with the volume
+func (d *driver) getScaleIoDevice(volumeID string) (string, error) {
+	_, err := d.GetVol(volumeID)
+	if err != nil {
+		return "", fmt.Errorf("Volume could not be located")
+	}
+
+	wwn := volumeID
+
+	devs, _ := ioutil.ReadDir(diskById)
+
+	for _, p := range devs {
+		name := strings.Replace(p.Name(), "-", "", -1)
+		if strings.Contains(name, wwn) {
```

Comment:
this seems like it could be error-prone, is it really contains, or is it StartsWith, etc?